containers/NodeJS/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,47 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/javascript-node
+{
+    "name": "Azure IoTEdge Node.js",
+    "build": {
+        "dockerfile": "Dockerfile",
+        // Update 'VARIANT' to pick a Node version: 12, 14, 16
+        "args": { "VARIANT": "16" }
+    },
+
+    "runArgs": [
+        "--network",
+        "host"
+    ],
+
+    "mounts": [
+        // Keep command history
+        "source=ostf-bashhistory,target=/commandhistory,type=volume",
+        // Use hosts docker socket
+        "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind"
+    ],
+
+    // Set *default* container specific settings.json values on container create.
+    "settings": {
+        "terminal.integrated.profiles.linux": {
+            "bash": {
+                "path": "/bin/bash"
+            }
+        }
+    },
+
+    // Add the IDs of extensions you want installed when the container is created.
+    "extensions": [
+        "dbaeumer.vscode-eslint",
+        "ms-vscode.azure-account",
+        "ms-azuretools.vscode-docker",
+        "vsciot-vscode.azure-iot-toolkit",
+        "vsciot-vscode.azure-iot-tools",
+        "vsciot-vscode.vscode-iot-device-cube"
```

Comment:
>vscode-iot-device-cube [](http://example.com/codeflow?start=23&length=22)

same comment here, this is already a dependency from the Workbench which is part of iot tools